front/src/modules/rollingStock/components/RollingStockCard/RollingStockCardDetail.tsx

```diff
@@ -167,11 +167,18 @@ export default function RollingStockCardDetail({
                 <td className="text-primary">{t('rollingResistanceB')}</td>
               </tr>
               <tr>
-                <td>
-                  {Math.floor(
+                <td
+                  title={
+                    rollingStock.rolling_resistance?.C
+                      ? ((rollingStock.rolling_resistance.C * 10000) / 10000).toString()
```

Comment:
Missing Math.floor.
if you want to have 5 decimals, you might as well use the function you just wrote

Reply:
We show the rounded value, but in the title we show the entire value

Reply:
I don't understand why you multiply by 1000 and then divide by 1000, am I missing smth ? I think it does nothing, you can remove the *1000/1000 if you remove the Math.floor

Reply:
yeah you can remove it I think